.github/workflows/cargo-features.yml

```diff
@@ -35,8 +35,8 @@ jobs:
       # as each individual feature enabled.
       - name: Check compilation for feature combinations
         run: |
-          cargo hack check --feature-powerset
+          cargo hack check --feature-powerset --exclude hotshot-query-service
```

Comment:
This is temporarily excluded because of the issue with dependencies in query service. We'll likely end up just merging these features into the parent sequencer module, but that'd require a high touch change. For now, we just remove it from the powerset because query service doesn't run correctly with certain subsets of features removed.